sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -178,6 +186,9 @@ def detect_language(  # type: ignore
         docs = _validate_input(documents, "country_hint", country_hint)
         model_version = kwargs.pop("model_version", None)
         show_stats = kwargs.pop("show_stats", False)
+        disable_service_logs = kwargs.pop("disable_service_logs", None)
+        if disable_service_logs:
```

Comment:
if `disable_service_logs=False` is passed this won't get added to kwargs. I think you want `if disable_service_logs is not None`